doc/news/changes/minor/20190209Arndt

```diff
@@ -0,0 +1,3 @@
+New: make_const_array_view creates a constant view from a non-const object.
```

Comment:
I think that for global functions, you need to write the name of the function plus `()` so doxygen recognizes it as a link.